mlrun/runtimes/nuclio/function.py

```diff
@@ -707,6 +707,12 @@ def enable_default_http_trigger(
         """
         self.spec.disable_default_http_trigger = False
 
+    def skip_base_image_enrichment(self):
+        # make sure the API does not enrich the base image with mlrun/mlrun if the function is not a python function
+        return (
+            self.spec.nuclio_runtime is None or "python" not in self.spec.nuclio_runtime
```

Comment:
when will nuclio_runtime be None?

Reply:
In most cases actually.
If it is not `python`, nuclio-jupyter enriches this field.
If it is python, we resolve the runtime in the server from the config:
https://github.com/mlrun/mlrun/blob/2bd77151e1c1a1051ddfd141cecc191cc79dbcc2/server/api/crud/runtimes/nuclio/function.py#L437-L442

Reply:
so if it is None and then in the server we resolve it to python. the image should be enriched no?

Reply:
Right I see it now.
I'll change it to:
```python
return self.spec.nuclio_runtime and "python" not in self.spec.nuclio_runtime
```